docker/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:22.04
```

Comment:
Why not 24.04?

Reply:
@ivanbaldo thanks for the review! I incorporated all your comments

Reply:
I should mention that the image size is now reduced by about .4GB, which is awesome